src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1568,26 +1615,34 @@ class ongoingTasksStats extends viewModelBase {
 
     private drawConnectionError(context: CanvasRenderingContext2D, x: number, y: number,
                                 errorType: Raven.Server.Documents.TcpHandlers.SubscriptionError) {
-        let dyForLine: number = 8;
-       
         let errorIcon: string;
         let iconStyle: string;
         
         if (errorType === "ConnectionRejected") {
             errorIcon = "\uea45";
             iconStyle = this.colors.tracks.ConnectionRejected;
-            dyForLine = 5;
+            this.drawErrorBackgound(context, iconStyle, x, y - 1);
         } else {
             errorIcon = "\uea44";
             iconStyle = this.colors.tracks.ConnectionAborted;
+            this.drawErrorBackgound(context, iconStyle, x, y - 1);
         }
-
-        context.strokeStyle = this.colors.tracks.ConnectionAborted;
-        graphHelper.drawLine(context, Math.round(x) + 0.5, y, dyForLine);
         
         context.fillStyle = iconStyle;
         context.font = "16px icomoon";
-        context.fillText(errorIcon, x - 8, y);
+        context.fillText(errorIcon, x - 8, y + 6);
+    }
+    
+    private drawErrorBackgound(context: CanvasRenderingContext2D, outlineColor: string, x: number, y: number): void {
```

Comment:
we should also make sure we don't draw that to often - will explain tomorrow on skype